applications/asset_tracker_v2/src/modules/sensor_module.c

```diff
@@ -192,19 +177,44 @@ static void ext_sensor_handler(const struct ext_sensor_evt *const evt)
 }
 #endif /* CONFIG_EXTERNAL_SENSORS */
 
+#if defined(CONFIG_EXTERNAL_SENSORS)
+static void configure_acc(const struct cloud_data_cfg *cfg)
+{
+		int err;
+		double accelerometer_activity_threshold =
+			cfg->accelerometer_activity_threshold;
+		double accelerometer_inactivity_threshold =
+			cfg->accelerometer_inactivity_threshold;
+		double accelerometer_inactivity_timeout =
+			cfg->accelerometer_inactivity_timeout;
+
+		err = ext_sensors_accelerometer_threshold_set(accelerometer_activity_threshold,
+							      true);
+		if (err == -ENOTSUP) {
+			LOG_WRN("Passed in act threshold value not valid");
```

Comment:
```suggestion
			LOG_WRN("The requested act threshold value not valid");
```
Same for other log messages below.